lib/Doctrine/Common/Cache/FileCache.php

```diff
@@ -124,11 +112,26 @@ public function getExtension()
      */
     protected function getFilename($id)
     {
+        if ('\\' === DIRECTORY_SEPARATOR) {
```

Comment:
@Tobion or we simply use this first block for everything (instead of `rawurlencode()`)